receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper.go

```diff
@@ -73,7 +73,7 @@ func (s *scraper) ScrapeMetrics(_ context.Context) (pdata.MetricSlice, error) {
 }
 
 func initializeCPUTimeMetric(metric pdata.Metric, startTime, now pdata.TimestampUnixNano, cpuTimes []cpu.TimesStat) {
-	metadata.Metrics.SystemCPUTime.CopyTo(metric)
+	metadata.MetricFactories.SystemCPUTime().CopyTo(metric)
```

Comment:
Hmm this results in 2 copies being done each time (one when SystemCPUTime() is called and another in the CopyTo). Hopefully this will be replaced by metricsbuilder though.

Reply:
Yeah it won't have that waste with the metric builder.